saleor/payment/gateways/utils.py

```diff
@@ -0,0 +1,22 @@
+import warnings
+from typing import TYPE_CHECKING, List
+
+from django.conf import settings
+
+if TYPE_CHECKING:
+    from ..interface import GatewayConfig
+
+
+def get_supported_currencies(config: "GatewayConfig", gateway_name: str) -> List[str]:
```

Comment:
Docstring would be useful here since this is a utility function that can be reused in more places.